### PR TITLE
Upgrade wp-prettier to 2.6.2 and reformat the codebase

### PR DESCRIPTION
**Pull request by frontdevde**
**tl;dr:** This PR aims to upgrade wp-prettier to 2.6.2 and reformats the codebase accordingly.

**Context:** While working on the block theme default styles project, I discovered a couple of files where the formatting was off and fixed that in a recent PR. I wondered how the formatting got through initially and, from there, went down a rabbit hole. I discovered several files with formatting/whitespace issues and initially started this PR to fix those. Then I noticed that in some areas, the formatting seemed off compared to what I had recently seen in Gutenberg and WooCommerce when working on the Elements API exploration...

Long story short, it turns out that the version of wp-prettier we're using in WooCommerce Blocks is quite a bit behind what Gutenberg and WooCommerce are using. I first wanted to bring the version up to par with what's being used there and then noticed that [jsnajdr](https://github.com/jsnajdr) recently created this PR https://github.com/WordPress/gutenberg/pull/40542 in Gutenberg to update wp-prettier to 2.6.2. It now got merged today.

> I updated the wp-prettier fork to be in sync with the latest upstream version, Prettier 2.6.2, and published it as wp-prettier@2.6.2. This PR updates the Gutenberg dependency and reformats the codebase.

So this PR does the same for WooCommerce Blocks.

### Testing

**Note:** Using the `Hide whitespace` option makes the Github Diff view more manageable. 

#### User Facing Testing

1. Ensure the project still builds as expected.
2. Ensure js linting returns no error.
3. Run `npm run reformat-files`. There shouldn't be any unformatted files.
4. Thoroughly smoke test the plugin to ensure everything is still working as expected.

* [x] Do not include in the Testing Notes 

### Changelog

> No changelog needed

### CODE REVIEW

**Review by Aljullu — APPROVED** [review]
LGTM, good catch, @frontdevde!
